src/NexusMods.App.UI/Windows/IWindowManager.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.ObjectModel;
+using System.Diagnostics.CodeAnalysis;
+using JetBrains.Annotations;
+
+namespace NexusMods.App.UI.Windows;
+
+/// <summary>
+/// Represents a manager of all existing windows.
```

Comment:
Add notes to comment: 
- We support multiple mainwindows. 
- Purpose for this is to help to identify the current window.